context/app/static/js/components/detailPage/provenance/ProvVis/ProvSchema.ts

```diff
@@ -0,0 +1,28 @@
+import { z } from 'zod';
+
+// Provenance JSON Schema converted to zod schema
+// Original schema came from https://github.com/hubmapconsortium-graveyard/prov-vis
+
+const stringToAnything = z.string().or(z.boolean()).or(z.number()).or(z.object({}).passthrough());
+const stringToStringToAnything = z.record(stringToAnything);
+
+export const provSchema = z
```

Comment:
Just commenting to note that we discussed you are copying the schema exactly as it was, but perhaps we could revisit this to make it more strict?

Reply:
I agree, the schema would be much more useful if it validated the actual types.
I created https://hms-dbmi.atlassian.net/browse/HMP-419 to handle this. 👍🏻